jenkins/kernel-trigger-tarball.sh

```diff
@@ -135,6 +135,16 @@ fi
 
 cd ${WORKSPACE}
 
+#
+# Dynamically create some special config fragments
+#
+# kselftests: create fragment by combining all the fragments from individual selftests
+#             fragment file will have comment lines showing which selftest dir
+#             each individual fragment came from
+#
+KSELFTEST_FRAG=kernel/configs/kselftest.config
```

Comment:
You need to do all this above the "cd ${WORKSPACE}" line, as you're no longer inside the kernel tree at this point. I made that change on kernelci-build-staging

https://github.com/kernelci/kernelci-build-staging/commit/e95f96442fe16fdaaf98535aae6f6d8fcfe546b1

In fact can you remake this pull request on kernelci-build-staging